lib/plugin/allure.js

```diff
@@ -283,7 +283,10 @@ module.exports = (config) => {
   let maxLevel;
   function finishMetastep(level) {
     const metaStepsToFinish = currentMetaStep.splice(maxLevel - level);
-    metaStepsToFinish.forEach(() => reporter.endStep('passed'));
+    metaStepsToFinish.forEach(() => {
+      // only if the current step is of type Step, end it. 
+      if (reporter.suites[0].currentStep.constructor.name === 'Step') { reporter.endStep('passed'); }
```

Comment:
sorry, I'm not sure that `reporter.suites[0].currentStep` will not throw exception by itself. Could you add a check that all of those objects do exist before calling them?

Reply:
Sure